DataConnectors/GCP/Terraform/sentinel_resources_creation/GCPInitialAuthenticationSetup/GCPInitialAuthenticationSetup.tf

```diff
@@ -11,7 +11,8 @@ terraform {
 
 locals{
   sentinel_app_id = "2041288c-b303-4ca0-9076-9612db3beeb2" // Do not change it. It's our Azure Active Directory app id that will be used for authentication with your project.
-  sentinel_tenant_id = "33e01921-4d64-4f8c-a055-5bdaffd5e33d" // Do not change it. It's our tenant id that will be used for authentication with your project.
+  sentinel_auth_id = "33e01921-4d64-4f8c-a055-5bdaffd5e33d" // Do not change it. It's our tenant id that will be used for authentication with your project.
+  workload_identity_pool_id = replace(var.tenant-id, "-", "") // Do not change it.
```

Comment:
Are we sure about the tenant id pattern? Can it have more chars?

Reply:
Also, is there any docs update to mention he must input the tenant-id? must he?

Reply:
Yes, it's in the docs of the onboarding. It also hasn't changed in this PR, customers already onboarded that way. It can't have more chars and that's ok. MDC also work in the same way